RedHatInsights/ros-frontend
# feat(RHIN-830): Add ZeroState to ROS

**Jira Card:** https://issues.redhat.com/browse/RHIROS-992

As in accordance to these mocks, this PR adds zero state to ROS,  https://docs.google.com/document/d/1FGuzbL2jyRmn_coYD5Z_o3d0MhOf1vQDT46NQvsfPyA/edit#heading=h.1hvxoje16w3i  https://www.sketch.com/s/550bd962-ca1f-42d8-8d89-d457527a059a/a/qbe9Ol2#Inspect   https://www.sketch.com/s/550bd962-ca1f-42d8-8d89-d457527a059a/prototype/a/AEF47CA9-8F00-40FC-B39B-3B989D3F83FD   To test, sign into an account with 0 systems connected 

The mocks mention using existing ROS logic but the logic I found was regarding permissions. Maybe someone with more experience on the app has a better understanding? However this should work as intended -- **shared information about the the same in the comment below**

(Added by Preeti)

### Steps to test
* Run `npm run start:beta`
* Test the new zero state UI with no user in the stage preview env

It looks like ATM the changes are only working in the stage-preview env, [check this thread for details](https://redhat-internal.slack.com/archives/C023VGW21NU/p1683218671958119).
